### PR TITLE
Write upgrade-guide for Plone 5.2

### PR DESCRIPTION
**Pull request by pbauer**

### CODE REVIEW

**Comment by stevepiercy** [comment]
@pbauer is this a WIP or do you want review?

**Comment by pbauer** [comment]
Feel free to review. There are still some TODOs but we can add them later.

**Review by stevepiercy — APPROVED** [review]
I just pushed a commit, instead of going through a review cycle.

This PR meets my approval as is and could be merged, but it wouldn't hurt to get another person's review so that I don't self-approve my changes.

File: manage/upgrading/version_specific_migration/upgrade_to_52.rst [hunk 1]
```diff
@@ -5,28 +5,315 @@ Upgrading Plone 5.1 to 5.2
...
+~~~~~~~~~~~~~
+
+Site-Administrators can use the URL Management control panel (``/@@redirection-controlpanel``) to manage and add alternative URLs.
+
+
+For Developers
+~~~~~~~~~~~~~~
+
+Since the add-on ``Products.RedirectionTool`` has been merged into Plone, you should remove it before the upgrade if you previously used it.
+
```
Comment (svx) [inline manage/upgrading/version_specific_migration/upgrade_to_52.rst:197]: Maybe I am stupid, but currently, I can not find any docs about ``Products.RedirectionTool`` in the docs branch 5.2.
We should make sure that we add these, I guess if I remember right there is also some UI involved, meaning pictures could not hurt :)

File: manage/upgrading/version_specific_migration/upgrade_to_52.rst [hunk 2]
```diff
@@ -5,28 +5,315 @@ Upgrading Plone 5.1 to 5.2
...
+.. code-block:: ini
+
+    [instance]
+    recipe = plone.recipe.zope2instance
+    eggs =
+        Plone[archetypes]
+        your.addon
+
+Instead of using Archetypes in Plone 5.2, you should consider migrating to Dexterity.
```
Comment (svx) [inline manage/upgrading/version_specific_migration/upgrade_to_52.rst]: Personally, I even set this as "note" directive to make that visual more prominent

File: manage/upgrading/version_specific_migration/upgrade_to_52.rst (hunk 1 above)
Comment (pbauer) [reply manage/upgrading/version_specific_migration/upgrade_to_52.rst:197]: This feature is visible as a new controlpanel that needs to be documented. The name ``Products.RedirectionTool`` should be mentioned but is no longer used. I think
@staeff and @mauritsvanrees did most of the work. Maybe they could document it.

File: manage/upgrading/version_specific_migration/upgrade_to_52.rst (hunk 2 above)
Comment (pbauer) [reply manage/upgrading/version_specific_migration/upgrade_to_52.rst]: ok

File: manage/upgrading/version_specific_migration/upgrade_to_52.rst (hunk 1 above)
Comment (mauritsvanrees) [reply manage/upgrading/version_specific_migration/upgrade_to_52.rst:197]: For documentation, see my [comment in the PLIP(https://github.com/plone/Products.CMFPlone/issues/1486#issuecomment-461637558) mentioning @svx. For now, that commit is the documentation. :-)

File: manage/upgrading/version_specific_migration/upgrade_to_52.rst (hunk 2 above)
Comment (pbauer) [reply manage/upgrading/version_specific_migration/upgrade_to_52.rst]: I did so

File: manage/upgrading/version_specific_migration/upgrade_to_52.rst (hunk 1 above)
Comment (pbauer) [reply manage/upgrading/version_specific_migration/upgrade_to_52.rst:197]: I added some of the text here.

**Comment by stevepiercy** [comment]
I've made additional revisions to @pbauer's commit. I still approve this PR, and it would still be good for another person to review so I don't self-approve my own commits.

**Comment by pbauer** [comment]
Thanks for the improvements! I'll merge.

**Pull request merged**